.github/ISSUE_TEMPLATE/Software-Project-Contribution.md

```diff
@@ -111,123 +122,36 @@ The [FINOS Technical Oversight Committee](https://github.com/finos/technical-ove
+- [ ] Review contribution checklist with FINOS staff
+- [ ] Review FINOS project blueprint
 
 # 4.  FINOS Onboarding
 
 This is performed by FINOS Infra once the three previous stages are complete, with support from the contributor and the FINOS Infra team.
 
-## Maintainers, Contributors and CLAs 
-- [ ] Identify other existing contributors (assuming there's a contribution history (eg Git history)
-- [ ] Check if maintainers and other contributors are all covered by FINOS CLA
-- [ ] Engage with FINOS Legal team to figure out what’s needed to cover all maintainers and contributors with FINOS CLA
-- [ ] Reach out to contributors and employers to coordinate CLA signatures
-- [ ] Invite GitHub usernames to GitHub FINOS Org
-- [ ] Create `<project-name>-maintainers` GitHub team and invite users
-
-## Validation (only if code is contributed)
-- [ ] Backup (even with screenshot) GitHub permissions of the repository to transfer
-- [ ] Check GitHub repository transfer requirements:
-  - [ ] [finos-admin](http://github.com/finos-admin) has `Admin` to all repositories to transfer
-  - [ ] [finos-admin](http://github.com/finos-admin) ia allowed to transfer repositories out of the org
-  - [ ] if the repository is owned by a user (and not an org), the user must be able to transfer the repository to [finos-admin](http://github.com/finos-admin)
-- [ ] The codebase doesn’t have HIGH or CRITICAL CVEs across direct and transitive libraries
-- [ ] The codebase doesn’t have any unfriendly licenses across direct and transitive libraries
-- [ ] (optional - if a build system is provided) The build process runs successfully
-- [ ] The codebase doesn’t include any patent or copyright that conflicts with FINOS Governance and bylaws (to be validated with FINOS Legal team)
-- [ ] [All incubating criteria](https://community.finos.org/docs/governance/Software-Projects/stages/incubating) are met (review documentation provided above)
-
-## Code transfer 
+## Code transfer preparation
+- [ ] Ensure [finos-admin](http://github.com/finos-admin) is Admin of the GitHub repository to transfer
+
+## Code Transfer Meeting
 - [ ] Transfer all code assets as GitHub repositories under github.com/finos
 - [ ] Rename main branch to `main` (instead of `master`)
-- [ ] Configure `finos-admins` (`Maintain` role) and `finos-staff` (`Triage` role) team permissions
-
-## Project Communication Channel(s)
-- [ ] Create the identified communication channels during infra set up
-- [ ] Create mailing-list on lists.finos.org (if requested)
-    - [ ] Enable Hubspot Sync for all project mailing lists created
-    - [ ] Aggregate mailing lists to community@finos.org
-    - [ ] Update marketing lists (see `Email List` and `Email` filter fields), particularly [Hubspot all community list](https://app.hubspot.com/contacts/2419532/lists/2247/filters)
-- [ ] Create Slack channels if requested.
-- [ ] Link communication channels linked front-and-center in the project README.md
-
-## Repository setup 
```

Comment:
Moved to Asana. Lines 201 + 203 are removed as hubspot sync does not work for [groups.io](http://groups.io/) mailing lists.